src/app/shell/personal-cabinet/parent/children/children.component.ts

```diff
@@ -34,7 +34,7 @@ export class ChildrenComponent extends CabinetDataComponent implements OnInit {
 
   init(): void {
     this.getUsersChildren();
-    this.getParentApplications();
+    this.getParentApplications('');
```

Comment:
you can make the parameter not required so you don't need to pass here anything

Reply:
done it according to our previous discussion about the possibility of implementing 'getParentApplications' by status backend team.